README.md

```diff
@@ -37,6 +37,15 @@ It looks for a string like `wire:model="email"` in your component's view file. I
 Livewire::test(FeedbackForm::class)
     ->assertMethodWired('submit');
 ```
+It looks for a string like `wire:click="submit"` in your component's view file. 
+
+### Check is a Livewire magic action is wired to an HTML field
```

Comment:
typo "is" to "if"

Reply:
Thanks a lot, looks great 🙌 Please fix the typo and then I think we are good to go 👍

Reply:
Typo fixed.